### PR TITLE
Adds About half of the Remaining LC Ego

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
Adds half the remaining LC ego, and a lot of the WL ones.
Also adds 5 points to WAW armor to make the examine text 100% accurate

image

## Why It's Good For The Game
We often have to wait for the ego to be coded after the abnormalities are done, while it sits there and no one wants to work on it due to it having no ego.
This fills out the ego for:
Bloodbath
Lady Facing the Wall
Singing Machine
Nameless Fetus
Snow Queen
Red Shoes
General B
Big and Will be Bad Wolf
Little Prince
Snow White's Apple
Nobody Is
Pygmalion

As well as a few background Wonderlabs armor for easy use later.

## Changelog
:cl:
add: Added plenty of new ego for upcoming abnormalities.
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/ego_gear/waw.dm [hunk 1]
```diff
@@ -1,10 +1,10 @@
-// WAW Armor should be kept at ~135 total armor.
+// WAW Armor should be kept at ~130 total armor.
```
Comment (EgorDinamit) [inline code/modules/clothing/suits/ego_gear/waw.dm:1]: Make it 140 instead, if you really need it.

**Review by EgorDinamit — COMMENTED** [review]
Make sure all the icons in the dmi are properly placed within their threat level.

File: code/modules/clothing/suits/ego_gear/waw.dm (hunk 1 above)
Comment (Kitsunemitsu) [reply code/modules/clothing/suits/ego_gear/waw.dm]: Done.

**Comment by EgorDinamit** [comment]
If all is done, rebase and squash commits.

**Pull request merged**